composer.json

```diff
@@ -37,11 +37,12 @@
     },
     "require-dev": {
         "ml/json-ld": "~1.0",
-        "phpunit/phpunit": "^7",
+        "phpunit/phpunit": "^7 || ^8",
         "code-lts/doctum": "^5",
         "semsol/arc2": "^2.4",
         "squizlabs/php_codesniffer": "3.*",
-        "zendframework/zend-http": "~2.3"
+        "zendframework/zend-http": "~2.3",
+        "yoast/phpunit-polyfills": "^1.0"
```

Comment:
This polyfill acts as a forward compatibility layer so tests running on PHPUnit 7 can use methods that only exist in PHPUnit 8.

Reply:
Great, that is really useful